tests/test_component/test_tmgr.py

```diff
@@ -27,6 +27,8 @@
 
 hostname = os.environ.get('RMQ_HOSTNAME', 'localhost')
 port     = int(os.environ.get('RMQ_PORT', 5672))
+username = os.environ.get('RMQ_USERNAME', 'guest')
+password = os.environ.get('RMQ_PASSWORD', 'guest')
 
```

Comment:
Is this preset in an ephemeral RBMQ we setup on Travis? If not, do we still use 'guest'?